ops/docker/profiler/profiler.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM node:16-bullseye
```

Comment:
Add  a comment warning that diat fails to compile on node-18. I've tried that and failed. Generally if we come back to this container we don't waste time trying bumping the node version.